### ISSUE TITLE
New helm-org repo

### ISSUE THREAD

**Issue by thierryvolpiatto** [emacs-helm/helm-org#1]
@alphapapa , @jagrg I have created a new repo dedicated for helm-org, you can commit here (or add PR).
Once helm-org.el ready to push to Melpa, I will remove helm-org.el from Helm.

Thanks.

**Comment by alphapapa** [comment]
For future reference, I will likely not notice issues and PRs posted to this repo very often, because GitHub provides no way to search for issues on both my personal repos and ones I belong to (there's no boolean OR in the search syntax).  Please tag me in anything that needs my attention, and I'll try to remember to check in periodically.

(I'm already "watching" the repo, but I'm also "watching" many other repos that I'm somewhat interested in, and there's no way to distinguish ones I'm responsible for other than "Participating", which is only the case on issues/PRs I've already interacted with, or using the advanced search, which, as I said, doesn't help here.)

Thanks.

**Comment by thierryvolpiatto** [comment]
alphapapa <notifications@github.com> writes:

> For future reference, I will likely not notice issues and PRs posted
> to this repo very often, because GitHub provides no way to search for
> issues on both my personal repos and ones I belong to (there's no
> boolean OR in the search syntax). Please tag me in anything that needs
> my attention, and I'll try to remember to check in periodically.

Don't you receive a mail for PR's and Issues as admin of helm-org?

Anyway I will tag you when needed.


--
Thierry

Get my Gnupg key:
gpg --keyserver pgp.mit.edu --recv-keys 59F29997

**Comment by alphapapa** [comment]
> Don't you receive a mail for PR's and Issues as admin of helm-org?

Yeah, but I get a deluge of email, especially from GitHub.  I guess I'll have to set up filters for repos I need to know about.  :)

**Comment by thierryvolpiatto** [comment]
alphapapa <notifications@github.com> writes:

>     Don't you receive a mail for PR's and Issues as admin of helm-org?
>    
> Yeah, but I get a deluge of email, especially from GitHub. I guess I'll have to set up filters for repos I need to know about. :)

Yes, it is what I do a filter for each github package I work on.

-- 
Thierry

Get my Gnupg key:
gpg --keyserver pgp.mit.edu --recv-keys 59F29997
